frontend/src/components/help-panel/TitleDescriptionHelpPanel.tsx

```diff
@@ -34,9 +34,15 @@ function TitleDescriptionHelpPanel({
           </h3>
           <ul>
             <li>
-              <a href={t('helpPanel.footer.docs.link')}>
+              <Link
+                external
+                externalIconAriaLabel={t(
```

Comment:
nitpick(non-blocking): this label is global and can be used every time there's the combination of `Link` and `external`. For this reason, I suggest to move this string into a "global" namespace and use it here like this:
```
t('global.openNewTab')
```